uaclient/tests/test_security_status.py

```diff
@@ -139,7 +141,7 @@ def test_finds_updates_for_installed_packages(
             mock_package(name="not_installed"),
             mock_package(
                 name="there_is_no_update",
-                installed=("1.0", "somewhere", "somehow", ""),
+                installed=("1.0", "somewhere", "Ubuntu", ""),
```

Comment:
Can we add a test with a component that is not `Ubuntu`, just to guarantee that the package does not enter the count

Reply:
There are a couple in this test - there are 5 installed packages, but only 2 of them are expected to be reported from `main` in the output (line 173 has an example)